provider/datagen/tests/test-options.rs

```diff
@@ -147,18 +274,18 @@ fn test_fallback_options() {
         "bn-u-nu-latn",
         "ccp",
         "ccp-u-nu-latn",
-        // "en", (same as 'und')
-        // "en-001", (same as 'und')
-        // "en-ZA", (same as 'und')
+        "en",
+        // "en-001", (same as 'en')
+        // "en-ZA", (same as 'en')
         "es",
         "es-AR",
-        // "fil", (same as 'und')
+        "fil",
         "fr",
-        // "ja", (same as 'und')
+        "ja",
         "ru",
         "sr", // Note: 'sr' and 'sr-Latn' are the same, but they don't inherit
         "sr-Latn",
-        // "th", (same as 'und')
+        "th",
```

Comment:
Issue: we should test that fallback to `und` works. You're making everything not be `und` which means we no longer test fallback to `und`.

Suggestion: just make `und` be the same as `en` and we can use the exact same cases as before.

Reply:
Is fallback to `und` different to all the other fallbacks?

I would want to trim the list of locales, we don't need this many to test that it works.

Reply:
`und` is different because

1. It is the last item in the fallback chain
2. `und` has lots of special handling code in both datagen and LocaleFallbacker and this is one opportunity to cover those special cases

Reply:
I recovered it